modules/nf-core/abricate/run/tests/main.nf.test

```diff
@@ -8,7 +8,7 @@ nextflow_process {
     tag "abricate"
     tag "abricate/run"
 
-    test("Should run without failures") {
+    test("bacteroides_fragilis - genome") {
```

Comment:
```suggestion
    test("bacteroides_fragilis - genome.fa.gz") {
```
This is a very small and not necessary change, but I personally like to see the file type for test datasets